dev-libs/libnl/files/libnl-3.11-rtnl_nh_get_oif.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/libnl-route-3.sym b/libnl-route-3.sym
```

Comment:
Absolutely not. We _cannot_ change ABI downstream.

Reply:
It means that a build against libnl in Gentoo will not be compatible with a build against _the same libnl version_ in another distribution.